### PR TITLE
fix build version to match semver

### PR DESCRIPTION
**Pull request by shalom938**

### CODE REVIEW

File: ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt [hunk 1]
```diff
@@ -228,15 +228,6 @@ class UpdatesService(private val project: Project) : Disposable {
         return appInfo.fullVersion
     }
 
-    // when plugin is not installed it will return 0.0.0
-    @NotNull
-    fun getPluginVersion(): String {
```
Comment (arik-dig) [inline ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt:233]: can you please bring this method back, and make the impl of it:
`SemanticVersionUtil.getPluginVersionWithoutBuildNumberAndPreRelease("0.0.0")`

**Review by arik-dig — APPROVED** [review]
one small comment - bring back the small method `getPluginVersion`

File: ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt (hunk 1 above)
Comment (shalom938) [reply ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt:233]: why?

File: ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt [hunk 2]
```diff
@@ -65,7 +64,7 @@ class UpdatesService(private val project: Project) : Disposable {
 
     init {
         stateBackendVersion = BackendVersionResponse(false, "0.0.1", "0.0.1", BackendDeploymentType.Unknown)
-        statePluginVersion = PluginVersion(getPluginVersion())
+        statePluginVersion = PluginVersion(SemanticVersionUtil.getPluginVersionWithoutBuildNumberAndPreRelease("0.0.0"))
```
Comment (arik-dig) [inline ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt]:
```suggestion
        statePluginVersion = PluginVersion(getPluginVersion())
```

File: ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt [hunk 3]
```diff
@@ -209,7 +208,8 @@ class UpdatesService(private val project: Project) : Disposable {
     @NotNull
     private fun buildVersionRequest(): VersionRequest {
         return VersionRequest(
-            getPluginVersion(), getPlatformType(), getPlatformVersion()
+            SemanticVersionUtil.getPluginVersionWithoutBuildNumberAndPreRelease("0.0.0"),
+            getPlatformType(), getPlatformVersion()
```
Comment (arik-dig) [inline ide-common/src/main/kotlin/org/digma/intellij/plugin/updates/UpdatesService.kt]:
```suggestion
getPluginVersion(), getPlatformType(), getPlatformVersion()
```

**Pull request merged**